src/app/projects/projlist-filters/projlist-filters.component.scss

```diff
@@ -317,29 +337,41 @@ $app-filters-font-lg: 0.9375rem;
 }
 
 .map-instructions, .project-type-filter {
-  margin: 0;
-  background: white;
-  padding: 0 0 1rem 1.5rem;
+  background: #fff;
+  padding: 2rem 0 0;
+
+  @media (max-width: 768px) {
+    padding: 0;
+  }
 }
 
-.project-type-filter-title-row {
+.project-type-filter-title-row, .project-search-title-row {
 	display: flex;
 	flex-direction: row;
 	width: 100%;
 	justify-content: space-between;
+  margin-bottom: 1.25rem;
```

Comment:
What happened to the spacing here?

Reply:
I'm not sure, it looks fine in my editor. I'll just use regular spaces to make sure it works.